http.go

```diff
@@ -228,13 +229,28 @@ func (h *RasterHttpServer) processParams(r *http.Request) (*RasterParams, int, e
 		return nil, 400, err
 	}
 
-	// Clean up the URL path into a local filename.
-	rParams.Filename = urlToFilename(r.URL.Path)
-	if rParams.Filename == "" || rParams.Filename == "/" {
+	// Clean up the URL path into a local file path.
+	if h.downloadManager == "dropbox" {
```

Comment:
I really don't like this solution. Can this be pushed all the way down into the Dropbox downloader? Embedding this logic here with the manager name directly smells to me.